src/Comet/EnvironmentData.cs

```diff
@@ -60,6 +60,11 @@ public static class View
 			public const string Opacity = nameof(Microsoft.Maui.IView.Opacity);
 		}
 
+		public static class Button
+		{
+			public const string Padding = "Padding";
```

Comment:
Why is this being added to Button? The Padding stuff already exists on layout, and should work for any view.

Reply:
That is MAUI button Padding property, I think it should be different from the padding of layout?